### PR TITLE
Add Doctrine 2 immutable datetime types to field mapping.

### PR DESCRIPTION
**Pull request by Sonny812**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| Doc updated   | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT

### CODE REVIEW

File: src/Metadata/Driver/AbstractDoctrineTypeDriver.php [hunk 1]
```diff
@@ -38,9 +38,13 @@ abstract class AbstractDoctrineTypeDriver implements DriverInterface
         'bigint' => 'integer',
 
         'datetime' => 'DateTime',
+        'datetime_immutable' => 'DateTime',
```
Comment (goetas) [inline src/Metadata/Driver/AbstractDoctrineTypeDriver.php]: Shouldn't this fields be mapped to `DateTimeImmutable` ?

Comment (Sonny812) [reply src/Metadata/Driver/AbstractDoctrineTypeDriver.php]: Sure. Looks good to me.

**Comment by Sonny812** [comment]
I think it looks cleaner.

**Comment by goetas** [comment]
Thanks!

**Pull request merged**